test/unit_tests/util/ObjectIOStreamTest.cpp

```diff
@@ -121,7 +121,7 @@ TEST(UtilObjectIOStream, testReadData) {
 TEST(UtilObjectIOStream, testReadImage) {
     // Generate a "random" image and serialize/deserialize it.
     std::mt19937 gen(4242);
-    std::uniform_int_distribution<unsigned char> distrib(0, 255);
+    std::uniform_int_distribution<uint16_t> distrib(0, 255);
```

Comment:
Yes, but it was required to even compile the tests on my machine, because std::uniform_int_distribution<char> seems to be a language extension, which is not supported by the standard / msvc STL. It might be better to just split that into a preceding commit.